torch/onnx/symbolic_opset11.py

```diff
@@ -180,6 +180,62 @@ def round(g, self):
     return g.op("Round", self)
 
 
+# Generate paddings in ONNX order based on pad in pytorch.
+# Arguments:
+#     dim: the dimension of the tensor.
+#     pad: the paddings in pytorch.
+#          The order is dim_n_begin, dim_n_end, dim_n-1_begin, dim_n-1_end, ...
```

Comment:
```suggestion
#          The order is dim_n_begin, dim_n_end, dim_n-1_begin, dim_n-1_end, ..., dim_m_begin, dim_m_end,
#          where m is in range [0, n].
```
The original comment was unclear that PyTorch padding could possibly omit zero paddings.